components/apimgt/org.wso2.carbon.apimgt.rest.api.commons/src/main/java/org/wso2/carbon/apimgt/rest/api/common/impl/BasicAuthAuthenticator.java

```diff
@@ -63,13 +63,13 @@ public boolean authenticate(Request request, Response responder, ServiceMethodIn
                     return true;
                 }
             } else {
-                throw new APIMgtSecurityException("Missing Authorization header in the request.`",
-                        ExceptionCodes.INVALID_AUTHORIZATION_HEADER);
+                throw new APIMgtSecurityException("Missing 'Authorization : Basic' parameter in the request.`",
```

Comment:
The 'parameter' word is interpreted from spec.

```
invalid_request
         The request is missing a required parameter, includes an
         unsupported parameter or parameter value, repeats the same
         parameter, uses more than one method for including an access
         token, or is otherwise malformed.  The resource server SHOULD
         respond with
```